java/code_to_gast/java_main.py

```diff
@@ -5,12 +5,18 @@
 takes js string and converts it to a generic AST
 """
 def java_to_gast(java_input):
-    # wrap input code in a class and function
-    class_wrapper = '''
-    class Test {{
-        public static void main(String[] args) {{
-            {java_input}
-        }}
-    }}'''.format(java_input = java_input)
-    input_ast = javalang.parse.parse(class_wrapper)
+    input_ast = ''
+
+    try:
```

Comment:
Is this wrapping needed if the caller (`main`) is also catching and will ignore the original error?

Reply:
From having played around with this- the first error that is caught is the one that is passed up. By having this here, we gain specificity about where the actual error is occurring.